pontoon/base/static/js/translate.js

```diff
@@ -469,18 +483,31 @@ var Pontoon = (function (my) {
      */
     openEditor: function (entity) {
       var self = this;
+      self.translationLengthLimit = null;
+
       $('#editor')[0].entity = entity;
 
-      // Metadata: comments, sources, keys
+      // Metadata: comment
       $('#metadata').empty();
       if (entity.comment) {
-        var comment = this.linkify(entity.comment);
-        if (comment === entity.comment) {
-          comment = this.doNotRender(entity.comment);
+
+        // Translation length limit
+        var split = entity.comment.split('\n'),
+            splitComment = entity.comment;
+        if (split[0].startsWith('MAX_LENGTH')) {
```

Comment:
We always put tags above comments, I checked all examples.

But you're right, tags are allowed to be positioned after the comment. We should catch this case when moving the logic to backend.